Navpage.html

```diff
@@ -0,0 +1,65 @@
+﻿<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+<!doctype html>
+<meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
+<link href="Navpagecss.css" rel="stylesheet" />
+<meta content="width=device-width, initial-scale=1" name="viewport" />
+<title>CookPal</title>
+</head>
+
+<body>
+
+
+<!--Nav contains collapsible sidebar with hamburger menu, title, and sign out button (if user is signed in)-->
+<!-- If the user is not signed in, another button will need to be added to sign in-->
+<nav>
+	<div id="sidebar" class="cSidebar">
+	    <button class="openButton" onclick="openNav()">☰</button>
+		<a href="#">About&nbsp;us</a> 
+		<a href="#">News</a> 
+		<a href="#">Support</a>
+	</div>
+	<div id="navbutton">
+		<button class="openButton" onclick="openNav()">☰</button></div>
+	<div id="navRest">
+        <h2 id="pageTitle">CookPal</h2>
+	</div>
+	<div>
+	<button id="signOutB">Sign&nbsp;out</button> 
```

Comment:
I think it's a SignIn not SignOut, after the user is signed in the navigation will display recipes and categories and other folders

Reply:
Good point, I will make the change now